frontend/packages/dev-console/src/models/pipelines.ts

```diff
@@ -10,6 +10,7 @@ export const PipelineModel: K8sKind = {
   kind: 'Pipeline',
   id: 'pipeline',
   labelPlural: 'Pipelines',
+  crd: true,
```

Comment:
Shouldn't the other pipeline models have `crd: true`?

Reply:
Added ```crd: true``` to all models.